omegatrade-app/backend/app/controller/company.controller.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const Company = require('../models/company.model')
+const { v4: uuidv4 } = require('uuid');
+
+/**
+ * Function to list all companies
+ * @method GET
+ */
+exports.getList = async function (req, res) {
+    try {
+        await Company.getAll(function (err, data) {
+            if (err)
+                return res.json({ success: false, message: "Error occured while fetching all Company" });
```

Comment:
nit: `Error occurred while fetching companies` (`...all Company` is not a user-friendly error message, as it seems to reference the function name instead of being a correct sentence)

Reply:
As mentioned in the below comments, All errors are changed to the genric message `something went wrong`.